core/src/main/java/org/opensearch/sql/planner/physical/PhysicalPlan.java

```diff
@@ -32,6 +32,14 @@ public void open() {
     getChild().forEach(PhysicalPlan::open);
   }
 
+  public String getCursor() {
```

Comment:
It breaks PhysicalPlan interface. Does it means our existing PhysicalPlan abstraction is not fit into cursor.